libraries/opensk/src/api/key_store.rs

```diff
@@ -81,6 +81,14 @@ pub trait KeyStore {
     /// The output is encrypted and authenticated. Since the wrapped credentials are passed to the
     /// relying party, the choice for credential wrapping impacts privacy. Looking at their size and
     /// structure, a relying party can guess the authenticator model that produced it.
+    ///
+    /// A credential ID that imitates the default needs the following structure:
+    /// - The length is `CBOR_CREDENTIAL_ID_SIZE`.
+    /// - The first byte is the version. The latest version is `CBOR_CREDENTIAL_ID_VERSION`.
```

Comment:
nit: You can make those links such that it's easy to navigate the doc.

```rust
    /// - The length is [`CBOR_CREDENTIAL_ID_SIZE`].
    /// - The first byte is the version. The latest version is [`CBOR_CREDENTIAL_ID_VERSION`].
```

(notice the square brackets)